lib/config/options/index.ts

```diff
@@ -2699,7 +2699,7 @@ const options: RenovateOptions[] = [
       ignoreDigest:
```

Comment:
I suspect that line 2698 needs to be changed as well, so that the `{{{newMajor}}}.x` part is also wrapped between backticks?

I hope/think this would work:

```diff
- 'Because you closed this PR without merging, Renovate will ignore this update. You will not get PRs for *any* future {{{newMajor}}}.x releases. But if you manually upgrade to {{{newMajor}}}.x then Renovate will re-enable `minor` and `patch` updates automatically.',
+ 'Because you closed this PR without merging, Renovate will ignore this update. You will not get PRs for *any* future `{{{newMajor}}}.x` releases. But if you manually upgrade to `{{{newMajor}}}.x` then Renovate will re-enable `minor` and `patch` updates automatically.',
```